### PR TITLE
(PC-12021) refactor(icons): replace category icons

### PR DESCRIPTION
**Pull request by yorickeando**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-12021

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [x] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [x] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

Capture d’écran 2021-12-02 à 09 43 29

Capture d’écran 2021-12-02 à 09 45 05

Capture d’écran 2021-12-02 à 09 45 09

Capture d’écran 2021-12-02 à 09 45 13

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/search/components/locationChoice.utils.tsx [hunk 1]
```diff
@@ -16,14 +16,14 @@ export const useLocationChoice = (
...
-      Icon: EverywhereDeprecated,
+      Icon: Everywhere,
       label: t`Partout`,
       isSelected: searchState.locationFilter.locationType === LocationType.EVERYWHERE,
     }
 
   if (section === LocationType.AROUND_ME)
     return {
-      Icon: AroundMeDeprecated,
+      Icon: AroundMe,
```
Comment (antoinewg) [inline src/features/search/components/locationChoice.utils.tsx:26]: 🥇

File: src/features/search/pages/Categories.tsx [hunk 2]
```diff
@@ -56,8 +56,8 @@ export const Categories: React.FC = () => {
               onPress={selectCategory(searchGroup)}
               testID={searchGroup}>
               <Spacer.Row numberOfSpaces={4} />
-              <Icon size={getSpacing(12)} color={ColorsEnum.PRIMARY} color2={color2} />
-              <Spacer.Row numberOfSpaces={2} />
+              <Icon size={getSpacing(10)} color={ColorsEnum.PRIMARY} color2={color2} />
```
Comment (antoinewg) [inline src/features/search/pages/Categories.tsx:59]: peut-être plus petit encore

File: src/features/search/pages/__tests__/LocationFilter.native.test.tsx [hunk 3]
```diff
@@ -104,6 +104,10 @@ describe('LocationFilter component', () => {
       venue: { ...Kourou, venueId: 4 },
     }
     const { queryByTestId } = renderLocationFilter()
+
+    // const component = renderLocationFilter()
+    // console.log(component)
```
Comment (antoinewg) [inline src/features/search/pages/__tests__/LocationFilter.native.test.tsx:109]: à supprimer si t'utilises pas

File: src/ui/svg/icons/BicolorLocationPointer.tsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ const NotMemoizedBicolorLocationPointer: React.FunctionComponent<IconInterface>
   size = 32,
   color = ColorsEnum.PRIMARY,
   color2 = ColorsEnum.SECONDARY,
-  testID,
+  testID = 'BicolorLocationPointer',
```
Comment (antoinewg) [inline src/ui/svg/icons/BicolorLocationPointer.tsx:13]: trop bien :)

**Review by antoinewg — APPROVED** [review]
trop bien. Merci pour le boulot 👏

File: src/features/search/pages/__tests__/LocationFilter.native.test.tsx (hunk 3 above)
Comment (yorickeando) [reply src/features/search/pages/__tests__/LocationFilter.native.test.tsx:109]: bien vu

**Pull request merged**